bedrock/firefox/templates/firefox/accounts-2018.html

```diff
@@ -0,0 +1,234 @@
+
+    <ul class="mobile-download-buttons mobile-pocket">
+      <li class="android">
+        {{ google_play_button(href='https://app.adjust.com/h5iycst', id='playStoreLink') }}
+      </li>
+      <li class="ios">
+        <a id="appStoreLink" href="https://app.adjust.com/h5iycst" data-link-type="download" data-download-os="iOS">
+          <img src="{{ l10n_img('firefox/ios/btn-app-store.svg') }}" alt="{{ _('Download on the App Store') }}" width="152" height="45">
+        </a>
+      </li>
+    </ul>
+  {% endcall %}
+
+  {% call feature_card(
+    title=_('Pick up where you left off with Notes'),
+    image_url='firefox/accounts/fxa-notes.png',
+    include_highres_image=True,
+    aspect_ratio='mzp-has-aspect-1-1',
+    class='mzp-l-card-feature-right-half notes-card',
+    link_cta=_('Get the Notes App'),
+    link_url='#'
+  ) %}
+    <p>{{ _('Your ideas and inspiration are secure and encrypted with Notes – and when you’re signed into your Account, they sync from your desktop to your Android devices.') }}</p>
+
+    <ul class="mobile-download-buttons mobile-notes">
+      <li class="android">
+        {{ google_play_button(href='https://play.google.com/store/apps/details?id=org.mozilla.testpilot.notes') }}
+      </li>
+    </ul>
+  {% endcall %}
+
+</div>
+
+<aside class="fxa-benefits mzp-u-modal-content">
+  <p class="mobile-title mobile-pocket">{{ _('Scan the QR Code to download the Pocket App on your mobile device.') }}</p>
+  <p class="mobile-title mobile-lockbox">{{ _('Scan the QR Code to download the Lockbox App on your mobile device.') }}</p>
+  <p class="mobile-title mobile-notes">{{ _('Scan the QR Code to download the Notes App on your mobile device.') }}</p>
+  <p id="fx-qr-title" class="mobile-title mobile-fx">{{ _('Scan the QR Code to download the Firefox App on your mobile device.') }}</p>
+  <div class="mzp-l-content">
+    <div id="qr-code"></div>
+  </div>
+
+  <section>
+    <div id="fx-app-buttons" class="show-fxa-supported-signed-in">
+        <ul class="mobile-download-buttons">
+          <li>
+            {{ google_play_button(id=android_id, extra_data_attributes={'download-location': button_location}) }}
+          </li>
+          <li>
+            <a href="{{ firefox_ios_url(app_store_campaign) }}" data-link-type="download" data-download-os="iOS" data-download-location="{{ button_location }}" id="{{ ios_id }}">
+              <img src="{{ l10n_img('firefox/ios/btn-app-store.svg') }}" alt="{{ _('Download on the App Store') }}" width="152" height="45">
+            </a>
+          </li>
+        </ul>
+      </div>
+  </section>
+
+  <section>
+    <ul class="mobile-download-buttons mobile-pocket">
+      <li class="android">
+        {{ google_play_button(href='https://app.adjust.com/h5iycst', id='playStoreLink') }}
+      </li>
+      <li class="ios">
+        <a id="appStoreLink" href="https://app.adjust.com/h5iycst" data-link-type="download" data-download-os="iOS">
```

Comment:
The id `appStoreLink` is used here and on line 71  and 161. IDs can only be used once on a  page.